src/api/test/test_helper.rb

```diff
@@ -155,15 +157,42 @@ def after_teardown
     #   exit
     # end
   end
+
+  def get(*args)
```

Comment:
This is _really_ ugly and I would rather not add this to our code base. Who knows what `ActionDispatch::Integration::RequestHelpers.get` will do/need/introduce in the future. This is such a basic method. Once shouldn't fiddle with it...